packages/select/src/Select/Select.spec.tsx

```diff
@@ -77,7 +77,7 @@ describe("<Select>", () => {
         render(
           <SelectWrapper
             selectedKey="id-sre"
-            isOpen
+            defaultOpen
```

Comment:
Updated this because the `isOpen` prop was firing the `onOpenChange` callback on mount